.github/workflows/prepare-release-pr.yml

```
@@ -0,0 +1,86 @@
+name: Prepare Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'The type of release to prepare a PR for.'
+        required: true
+        type: choice
+        default: 'stable'
+        options:
+          - stable
+          - alpha
+          - beta
+          - rc
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  prepare_release_pr:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          fetch-depth: 0
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+
+      - name: Set node version to 22
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 22
+          cache: 'pnpm'
+
+      - name: Install deps
+        run: pnpm install
+        env:
+          CYPRESS_INSTALL_BINARY: 0
+
+      - name: Run release
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
```

Alternatively:

```suggestion
          git config user.name "${GITHUB_ACTOR}"
          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
```

or

```suggestion
          git config user.name "${{ github.actor }}"
          git config user.email "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
```

(not tested)

---

This is only semantically or does this imply actual changes during runtime? As far as I can see it would be the first one. I want to make sure that I got this correct.

---

AFAICT this doesn't change anything, except the username being associated with the commit.